pkg/cloud/services/powervs/service.go

```diff
@@ -110,6 +111,16 @@ func (s *Service) GetAllNetwork() (*models.Networks, error) {
 	return s.networkClient.GetAll()
 }
 
+// GetDHCPServers returns all the DHCP servers in the Power VS service instance.
+func (s *Service) GetDHCPServers() (models.DHCPServers, error) {
```

Comment:
these functions are named like <client_function><client_name>, so this can be renamed as
```suggestion
func (s *Service) GetAllDHCPServers() (models.DHCPServers, error) {
```